brion/csvConfig.h

```diff
@@ -42,10 +42,10 @@ class CsvConfig
     CsvConfig(const CsvConfig& other) = delete;
     ~CsvConfig();
 
-    /** Returns all available node_type_id's */
+    /** Returns all available node_type_id's in random order */
```

Comment:
It's not random because it doesn't change everytime you call it, it's just undefined